tests/functional/test_basic.py

```diff
@@ -545,3 +545,26 @@ def test_hook_collect_submodules(pyi_builder, script_dir):
 # Test that PyInstaller can handle a script with an arbitrary extension.
 def test_arbitrary_ext(pyi_builder):
     pyi_builder.test_script('pyi_arbitrary_ext.foo')
+    
+def test_option_runtime_tmpdir(pyi_builder):
+    "Test to ensure that option `runtime_tmpdir` can be set and has effect."
+
+    pyi_builder.test_source(
+        """
+        print('test - runtime_tmpdir - custom runtime temporary directory')
+        import os
+        import sys
+        if sys.platform == 'win32':
+            import win32api
+        cwd = os.path.abspath(os.getcwd())
+        if sys.platform == 'win32' and sys.version_info.major == 2 and sys.version_info.minor == 7:
```

Comment:
We only support 2.7 and 2.x. So this can be shortened to  `… and sys.version_info < (3,):`